CondTools/SiPixel/plugins/SiPixelGenErrorDBObjectReader.h

```diff
@@ -4,27 +4,30 @@
 #include <memory>
 
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/ESWatcher.h"
 
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
 
 #include "CondFormats/SiPixelObjects/interface/SiPixelGenErrorDBObject.h"
 #include "CondFormats/DataRecord/interface/SiPixelGenErrorDBObjectRcd.h"
 #include "CalibTracker/Records/interface/SiPixelGenErrorDBObjectESProducerRcd.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
 
-class SiPixelGenErrorDBObjectReader : public edm::EDAnalyzer {
+class SiPixelGenErrorDBObjectReader : public edm::one::EDAnalyzer<> {
 public:
   explicit SiPixelGenErrorDBObjectReader(const edm::ParameterSet&);
-  ~SiPixelGenErrorDBObjectReader();
+  ~SiPixelGenErrorDBObjectReader() override;
 
 private:
-  virtual void beginJob();
-  virtual void analyze(const edm::Event&, const edm::EventSetup&);
-  virtual void endJob();
+  void beginJob() override;
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+  void endJob() override;
 
   //edm::ESWatcher<SiPixelGenErrorDBObjectESProducerRcd>  SiPixGenerDBObjectWatcher_;
```

Comment:
I see no hint of these variables, even commented out, in the implementation code: they can probably be safely removed also here.